functions/exercise2/scriptSpec.js

```diff
@@ -1 +1,25 @@
 /// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals checkPalindrome*/
```

Comment:
checkPalindrome is not the best name here as it does not state what is expected returned type here.
It is better to call it isPalindrome which is more obvious for function returning value of boolean type.